awx/api/serializers.py

```diff
@@ -3204,7 +3204,7 @@ def build_standard_field(self, field_name, model_field):
             field_kwargs['choices'] = module_name_choices
             field_kwargs['required'] = bool(not module_name_default)
             field_kwargs['default'] = module_name_default or serializers.empty
-            field_kwargs['allow_blank'] = bool(module_name_default)
+            field_kwargs['allow_blank'] = False
```

Comment:
My rationale for this change is that you really do need a module name to run an AHC. It also fixes this:

![Screenshot from 2020-03-25 11-37-09](https://user-images.githubusercontent.com/9753817/77559673-d2d9ef80-6e92-11ea-80ed-c8a030a5d32d.png)